recipes/halotools/meta.yaml

```diff
@@ -0,0 +1,106 @@
+{% set name = "halotools" %}
+{% set version = "0.5" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "7f5e8c3afcee35baa053a2c084d1b1301de70a285fce2391b7e46fedb53e041d" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --offline --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy 1.9.*  # [not (win and py36)]
```

Comment:
Once I get all of the astropy recipes building on conda-forge the plan is to use the astropy channel repo to run the test suites of these packages to make sure the astropy channel is consistent with defaults and with conda-forge. If we include a couple of numpy versions we should be able to catch any errors.

That is likely a month away, but who knows...it could happen sooner!